flink-scala-shell/src/test/scala/org/apache/flink/api/scala/ScalaShellITCase.scala

```diff
@@ -460,6 +460,7 @@ object ScalaShellITCase {
 
   val _clusterResource = new MiniClusterResource(new MiniClusterResourceConfiguration.Builder()
       .setNumberSlotsPerTaskManager(parallelism)
+      .disallowChangelogState()
```

Comment:
(Optional) 
I think it would be helpful to add (as a comment) the reason why it is disallowed in each case.